stable/anchore-admission-controller/templates/webhook.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.apiService.webhook.enabled }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: {{ template "anchore-admission-controller.name" . }}-anchore-admission-controller.admission.anchore.io
+labels:
+  app.kubernetes.io/name: {{ template "name" . }}
+  helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version \| replace "+" "_" }}
+  app.kubernetes.io/managed-by: {{ .Release.Service }}
+  app.kubernetes.io/instance: {{ .Release.Name }}
+  app.kubernetes.io/version: {{ .Chart.AppVersion }}
+webhooks:
+- name: {{ template "anchore-admission-controller.name" . }}-anchore-admission-controller.admission.anchore.io
+  clientConfig:
+    service:
+      namespace: default
+      name: kubernetes
+      path: /apis/(( .Values.apiService.group ))/v1beta1/imagechecks
+    caBundle: {{ .Values.apiService.webhook.caBundle | b64enc }}
+  rules:
+  - operations:
+    - CREATE
+    apiGroups:
+    - ""
+    apiVersions:
+    - "*"
+    resources:
+    - pods
+  failurePolicy: {{ .Values.apiService.webhook.failurePolicy }}
+  namespaceSelector:
```

Comment:
For this, how about adding a section in the values.yaml that lets a user insert a selector section verbatim. That would allow support for other types of selectors as well. Maybe a key 'selectors' that has its content added into the webhook object.